### PR TITLE
tests: stablize test_download_sst, test_invalid_read_index_when_no_leader

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

###  What have you changed?

Fix test_download_sst, test_invalid_read_index_when_no_leader
https://github.com/tikv/tikv/issues/7146 https://github.com/tikv/tikv/issues/7130

###  What is the type of the changes?

- Bugfix (a change which fixes an issue)

###  How is the PR tested?

- Unit test

### CODE REVIEW

File: tests/integrations/import/sst_service.rs [hunk 1]
```diff
@@ -137,10 +137,7 @@ fn test_download_sst() {
 
     let result = import.download(&download).unwrap();
     assert!(
-        result
-            .get_error()
-            .get_message()
-            .contains("CannotReadExternalStorage"),
```
Comment (overvenus) [inline tests/integrations/import/sst_service.rs]: Why is it not `"CannotReadExternalStorage"`?

Comment (5kbpers) [reply tests/integrations/import/sst_service.rs]: Actually the error message is `Cannot read ...`, I'm confused why it can be passed for the first time.

Comment (youjiali1995) [reply tests/integrations/import/sst_service.rs]: I found it returns
```
test import::sst_service::test_download_sst ... thread 'main' panicked at 'unexpected download reply: error { message: "Cannot read local:///tmp/test_download_sst9LnKJl/missing.sst" }', tests/integrations/import/sst_service.rs:139:5
```

Comment (5kbpers) [reply tests/integrations/import/sst_service.rs]: Seems the original error message is printed in `debug` mode and now is printed in `display` mode.😓

**Pull request merged**